custom_components/entsoe/const.py

```diff
@@ -134,14 +136,38 @@ class EntsoeEntityDescription(SensorEntityDescription):
     ),
     EntsoeEntityDescription(
         key="highest_price_time_today",
-        name="Time of highest price today",
+        name="Entsoe time of highest price today",
```

Comment:
Please don't change the name property. 
It is used in the entity naming and therefore, if this were to be merged, all history, dashboards and automations will fail for end-users

Reply:
If you personally want to have entsoe in the name then you should set that in the name configuration field (the configuration option name gets pre-apended to the entity name to allow for multiple integration instances)

Reply:
Yes, that has been changed back after I rebased to the JaccoR's top.